### PR TITLE
Add support for nwm::EXT::ControlWirelessEnabled.

### PR DESCRIPTION
**Pull request by thedax**
https://www.3dbrew.org/wiki/NWMEXT:ControlWirelessEnabled

### CODE REVIEW

File: libctru/include/3ds/services/nwmext.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Result nwmExtInit(void);
+
+// Exits NWMEXT.
+void nwmExtExit(void);
+
+/**
+ * @brief Turns wireless on or off.
+ * @param command Command to turn wireless on or off.
+ */
+Result NWMEXT_ControlWirelessEnabled(u32 command);
```
Comment (fincs) [inline libctru/include/3ds/services/nwmext.h]: Why isn't this just a bool?

Comment (thedax) [inline libctru/include/3ds/services/nwmext.h]: As far as I can tell, just about all of the other services use Results, so why should this be any different? When I tested it a while back it was able to return error codes, not just a true or false boolean value.

Comment (fincs) [inline libctru/include/3ds/services/nwmext.h]: I mean the input parameter, not the output parameter.

Comment (thedax) [inline libctru/include/3ds/services/nwmext.h]: Oh, yeah. Sorry, I misunderstood. I can change that no problem.

**Comment by thedax** [comment]
Rebased and squashed.

**Pull request merged**